samples/Calling/src/app/utils/AppUtils.ts

```diff
@@ -41,20 +41,12 @@ export const getGroupIdFromUrl = (): GroupLocator | undefined => {
 
 export const createGroupId = (): GroupLocator => ({ groupId: generateGUID() });
 
-/* @conditional-compile-remove(rooms) */
-export const ROOMS_API_BASE_URL = 'http://localhost:7071';
-
 /* @conditional-compile-remove(rooms) */
 export const createRoomId = async (): Promise<string> => {
   const requestOptions = {
-    method: 'POST',
-    headers: {
-      Host: 'alkwa-fn-test.azurewebsites.net',
-      'Content-Type': 'application/json',
-      'Referrer-Policy': 'no-referrer'
-    }
+    method: 'POST'
   };
-  const response = await fetch(ROOMS_API_BASE_URL + '/api/Rooms-CreateRoom', requestOptions);
+  const response = await fetch(`/createRoom`, requestOptions);
   if (!response.ok) {
```

Comment:
invalid token response doesn't seem appropriate for creating a room

"Unable to create room" might be better

Reply:
Changed. Good call.